dwh/query_lib_big_query.py

```diff
@@ -201,15 +206,12 @@ def _all_obs_constraints(self) -> str:
 
     def get_patient_encounter_view(
```

Comment:
Actually, as we discussed earlier, the `base_url` in the spark implementation has a default `empty` string now. I believe as part of another PR you will be getting rid of that parameter. Therefore, I went ahead and removed it in this class.

Reply:
Right, but my suggestion was to actually remove `base_url` in the Spark version as well. If you feel we should leave it for another PR, I strongly suggest that we at least fix this inheritance issue. I can think of at least two ways to do this without touching Spark code much:
- Add a `**kwargs` parameter to the base and remove `base_url` from the signature. I think this way at least all implementations are overriding the base funciton.
- Remove `base_url` from the signature but add a `base_url = ''` parameter in the Spark implementations as the first line (so that nothing else need to be changed).
The current form is confusing and is also causing some other issues which I commented about, e.g., in tests.

Reply:
To be clear, if you go with the second option (which is my preference) you need to update where `get_patient_*_view` is called and drop `base_url`. A quick check shows that there are currently 4 usages in `indicators.py` and 3 in `test_query_lib.ipynb`